src/app/shared/layout/header.component.ts

```diff
@@ -37,6 +39,15 @@ export class HeaderComponent implements OnInit {
     }
   }
 
+  viewBrowser() {
+    const repoUrl = this.githubService.getRepoURL();
+    let issueUrl = this.router.url.substring(7); // remove the '/phase' from string
+    if (issueUrl === '') {
+      issueUrl = '/issues';
+    }
+    shell.openExternal('https://github.com/'.concat(repoUrl).concat(issueUrl));
```

Comment:
If users were to click from the issues-list page, would it be better to redirect users to an already filtered list issues in Github?

Reply:
I have added filter value to the url but Github filter doesnt really work for incomplete keyword.

Reply:
Hmm, how about filter by team(for students)/teams(for tutor/admin)? 
@damithc , do you think it would be good to redirect the user to a filtered list of issues on Github?

Reply:
Ideally, the view should match what CATcher shows. But if that is not possible, it can be something close to it.

Reply:
I have added the team filter to the url. It should work for students/tutor/admin